packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -167,9 +169,131 @@ test.serial('stakeAtom - smart wallet', async t => {
   t.like(wd.getLatestUpdateRecord(), {
     status: { id: 'request-account', numWantsSatisfied: 1 },
   });
-
-  const { ATOM } = agoricNamesRemotes.brand;
+  const { ATOM, DAI_axl, BLD } = agoricNamesRemotes.brand;
   ATOM || Fail`ATOM missing from agoricNames`;
+  DAI_axl || Fail`DAI_axl missing from agoricNames`;
+  BLD || Fail`BLD missing from agoricNames`;
+
+  // 2. Deposit to Account
+  await wd.executeOffer({
+    id: 'request-deposit-success',
+    invitationSpec: {
+      source: 'continuing',
+      previousOffer: 'request-account',
+      invitationMakerName: 'Deposit',
+    },
+    proposal: {
+      give: {
+        // @ts-expect-error BoardRemote is not assignable to Brand<any>
```

Comment:
I/we really should fix that. the `getBoardId()` kludge is a bad idea. the mapping from value to slot (board id) belongs in a table external to the object; we learned this in [makeClientMarshaller](https://github.com/Agoric/ui-kit/blob/main/packages/rpc/src/marshal.ts#L37) but we haven't back-ported it all the way to here.

IOU an issue, I guess.